src/objective-c/ProtoRPC/ProtoMethod.h

```diff
@@ -48,3 +49,11 @@
                         service:(NSString *)service
                          method:(NSString *)method;
 @end
+
+/**
+ * This subclass is empty now. Eventually we'll remove ProtoService class
+ * to avoid potential naming conflict
```

Comment:
nit: remove `the` ProtoService class [...] conflict`s.`

Same in the docs of the other two.